### DISCUSSION TITLE
vercel --prod hangs forevers

### DISCUSSION THREAD

**Discussion by danilockthar** [vercel/vercel#4726] [Help] [answered]
Im deploying to an existent project and the vercel --prod commands hangs on queued loading forever. First time it happens. Any idea ?

**Comment by styfle** [comment, marked as answer]
Hi @danilockthar 

We are currently investigating elevated build queue processing times for a small number of customers. Existing deployments and live traffic are not affected by this issue. We will share updates as they become available at https://www.vercel-status.com

Sorry about the delay!

**Reply by danilockthar** [reply]
Thank you!
